eng/pipeline/stages/run-stage.yml

```diff
@@ -124,16 +138,21 @@ stages:
               - pwsh: Write-Host "##vso[task.setvariable variable=GOARM]6"
                 displayName: Set GOARM for cross-compile
 
+          - ${{ if eq(variables.createPDB, true) }}:
```

Comment:
Interesting... wasn't aware job variables would be accessible here. (I experienced pipeline variables *not* being accessible inside a stage template in go-images, but it seems now that that's because that stage's variables are the ones that are in scope at that point, excluding the outer variables.)

https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#variables seems very vague about it, but another thing it mentions is that they're evaluated *in order*... so maybe some of the value reuse we're currently doing through layers of templates could be done this way too.

No suggestion to change anything now, just some context about why this isn't showing up anywhere else. 😄